### ISSUE TITLE
Sorting destructured function props?

### ISSUE THREAD

**Issue by ajbogh** [mthadley/eslint-plugin-sort-destructure-keys#113]
Does this plugin also sort destructured function props? I've attempted to use it and it wasn't flagging or fixing the error. I just wanted to verify that it's supposed to be a feature or if it's missing.

Example:
```
const MyComponent: React.FC<Props> = ({
  children,
  theme = darkTheme, // should be below "isOpen", not marked in error, not fixed
  isOpen = false,
  useThing = true
}): React.ReactElement => (
```

**Comment by mthadley** [comment]
It does, but the plugin won't try to sort in that example due to the `darkTheme` default. See #18.

(But I also haven't used this with Typescript, so I don't know if there's other reasons...)
